### PR TITLE
{perf}[intel/2015a] OSU Micro-Benchmarks v4.4.1 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
@besserox: any thoughts on this?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/931f0c4f24b0c9ad2595 for a full test report.

**Comment by besserox** [comment]
It looks good. 

However, there might be the same problem as MVAPICH2 in the future, ie the source package of the old versions might disappear from the website.

**Comment by boegel** [comment]
@besserox: possibly, but there's nothing much we can do about that...

File: easybuild/easyconfigs/o/OSU-Micro-Benchmarks/OSU-Micro-Benchmarks-4.4.1-intel-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+name = 'OSU-Micro-Benchmarks'
+version = '4.4.1'
+
+homepage = 'http://mvapich.cse.ohio-state.edu/benchmarks/'
+description = """OSU Micro-Benchmarks"""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+source_urls = ['http://mvapich.cse.ohio-state.edu/download/mvapich/']
+sources = ['%(namelower)s-%(version)s.tar.gz']
```
Comment (wpoely86) [inline easybuild/easyconfigs/o/OSU-Micro-Benchmarks/OSU-Micro-Benchmarks-4.4.1-intel-2015a.eb]: `SOURCELOWER_TAR_GZ`?

File: easybuild/easyconfigs/o/OSU-Micro-Benchmarks/OSU-Micro-Benchmarks-4.4.1-intel-2015a.eb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+configopts = 'CC="$MPICC"'
+
+modextrapaths = {'PATH': ['libexec/osu-micro-benchmarks/mpi/%s' % x for x in ['collective', 'one-sided', 'pt2pt']]}
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['libexec/osu-micro-benchmarks/mpi/%s' % x for x in ['collective', 'one-sided', 'pt2pt']],
```
Comment (wpoely86) [inline easybuild/easyconfigs/o/OSU-Micro-Benchmarks/OSU-Micro-Benchmarks-4.4.1-intel-2015a.eb]: define the array only once?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/608ad50193b1ca59b1bf for a full test report.

**Comment by boegel** [comment]
Going in, thanks for the review @besserox and @wpoely86!

**Pull request merged**